src/modules/wallet/components/PassphraseInput/PassphraseInput.test.js

```diff
@@ -99,7 +99,8 @@ describe('passphraseInput', () => {
       expect(props.onFill).toBeCalledWith(passphrase, 'Required');
     });
 
-    const ONLY_ONE_WORD_ERROR = 'Passphrase should have 12 words, entered passphrase has 1';
+    const ONLY_ONE_WORD_ERROR =
+      'Passphrase contains 1 words instead of expected 12. Please check the passphrase.';
```

Comment:
```suggestion
      'Passphrase contains 1 word instead of expected 12 words. Please check the passphrase.';
```

Reply:
This is how SDK returns the error message, so we can not change it :(

Reply:
ohh okay